hibernate-jipijapa/hibernate-jipijapa.gradle

```diff
@@ -0,0 +1,22 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+apply from: rootProject.file( 'gradle/published-java-module.gradle' )
+
+description = 'Integrate with WildFly JipiJapa '
+
+dependencies {
+    compile project( ':hibernate-core' )
+    compile( "org.wildfly:jipijapa-spi:12.0.0.Final" )
+    compile( "org.jboss:jboss-vfs:3.2.11.Final" )
+    // compile( "org.infinispan:infinispan-core:9.2.0.Final" )
+    compile( "org.infinispan:infinispan-hibernate-cache-commons:9.2.0.Final" )
```

Comment:
this should probably use the `infinispanVersion` property?

Reply:
sure, it was easier for me to just hard code the versions in my first .gradle file, I'll fix this. :)